Gruntfile.js

```diff
@@ -0,0 +1,42 @@
+/*
+ * gpii-grunt-mdjson-linter
```

Comment:
Update for current package name

Reply:
Fixed here and in the package.json as well.